### PR TITLE
Move Rococo detials to HTG 

### PR DESCRIPTION
**Pull request by NukeManDan**
I also crosslink the cumulus tutorial, being a pre-req to the Rococo HTG. Thus we _don't repeat ourselves_ on general parachain instuctions and basics. 

## TODO
- [ ] Get buyin from webdev team on design needs
- [ ] Check content is correct
- [ ] Update the Polkadot wiki to remove all instructions at direct to this HTG for Rococo. Crosslink.
- [ ] Crosslink to any substrate.io page on Rococo.

cc @al3mart @SBalaguer @BenWhiteJam @rmnprkrl

### CODE REVIEW

**Comment by BenWhiteJam** [comment]
Don't we need to add the whole paraID stuff, like reserve paraID, build wasm from paraID etc.?

**Comment by sacha-l** [comment]
@BenWhiteJam : the assumption is that at this stage, individuals or teams already have their genesis state and wasm blob. 

@NukeManDan : I've pushed a commit to fix the content to make it more like a guide. Please have a look and make sure it renders correctly (checking for formatting _as rendered, not in theory_  ;))  . Also ported over the example from #983. 

As per the todo checklist, I don't see any of those items being blockers to this content being updated this way. Polkadot wiki and pages external to our docs are concerns we should address separately.

**Comment by sacha-l** [comment]
We're also missing details on adding endpoints and favicons for team. @NukeManDan could you please complete this guide to include these steps? (e.g. https://github.com/polkadot-js/apps/pull/7349/files) 

@BenWhiteJam 
What I intend with this is that the "marketing Rococo page" becomes purely an entry point to this guide. It shouldn't have any technical steps at all. 

If anything it could contain steps like:

1. Get ROC tokens (link to guide: "Preparation")
1. Register as a parathread (link to guide: "Register as a parathread")
1. Submit your slot request (link to guide: "Request your slot")
1. You're ready to go.

I think the page should house top-level information on how it works, such as the Examples I added in my last commit. It should tell a story (perhaps with the existing graphics). But certainly not have code snippets just to look pretty.

**Comment by al3mart** [comment]
Having that example of how temp slots rotate can be a good way of showing how assinged_slots works and setting some expectations for teams.

There is something that is not covered in the example, and I think it should not be included for not making it too harsh, but the pallet take into account for onboarding paras that haven't been onboarded for 2x the duration of the temp lease. So it avoids sarvation.

**Comment by NukeManDan** [comment]
@sacha-l  - Can more steps be added, yes! A follow up PR can be made for this though, so we get the structure of the HTG up ASAP and add to it. 

@al3mart would you be able to open a PR to add this detail on assigned slot mechanics and process? I am happy to review.

**Comment by al3mart** [comment]
> add this detail on assigned slot mechanics and process?

@NukeManDan Where is this ? :sweat_smile:

File: v3/how-to-guides/08-parachains/f-rococo/index.mdx
Comment (NukeManDan) [inline v3/how-to-guides/08-parachains/f-rococo/index.mdx:162]: @al3mart here is the basics of the slot detials, but more about the actual logic could be useful if you see fit. If you think this is detailed enough, then OK. We can do a follow up PR if you like as well. I would think we are good to approve once you or Sacha give a final pass. Feel free to squash merge this PR once approved

Comment (al3mart) [reply v3/how-to-guides/08-parachains/f-rococo/index.mdx]: As I see changes coming to this precise pallet I would say following with a PR could be more handy.

**Review by al3mart — APPROVED** [review]

**Review by sacha-l — APPROVED** [review]

**Pull request merged**
